dialogue-serde/src/main/java/com/palantir/conjure/java/dialogue/serde/ErrorDecoder.java

```diff
@@ -51,17 +51,17 @@
  * {@link RemoteException}) if a {@link RemoteException} could not be extracted, e.g., when the given {@link
  * Response} does not adhere to an expected format.
  */
-enum ErrorDecoder {
+public enum ErrorDecoder {
```

Comment:
This becomes public API, potential downside is that it's harder to add additional info (metrics?) to this type in the future, but I think it's more important that we allow everything to parse/throw the correct exceptions. Worst case we can always gut this enum and move the logic elsewhere, leaving this class for backcompat.